baselines/bigquery-storage/.eslintrc.json.baseline

```diff
@@ -0,0 +1,34 @@
+{
+  "extends": "./node_modules/gts",
+  "rules": {
```

Comment:
I don't think we want any of these rules here .... do we?  For each of these, we should rationalize them in `gts`.

Reply:
added extra rules to `gts` and hold this to use the latest gts version.